.github/workflows/vscode.yml

```diff
@@ -0,0 +1,70 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - main
+      - release/*
+  pull_request:
+    branches:
+      - main
+      - release/*
+jobs:
+   vscode_azure_iot_edge_Stage_Linux:
+    name: Linux
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Install NodeJs
+      uses: actions/setup-node@v1
+      with:
+       node-version: 16.x
+    - name: npm install
+      run: |
+         npm install
+         sudo npm install -g vsce
+    - name: npm Package
+      run: |
+         npm run tslint
+         npm run compile
+         node scripts/genAiKey.js
+         echo "y" | vsce package
+    - name: npm test
+      run: xvfb-run --auto-servernum --server-args="-screen 0, 1024x768x24" npm test --silent
+    - name: docker test
+      run: docker run -t --rm -v $PWD:/mnt:ro dkhamsing/awesome_bot --white-list "gitter.im/Microsoft/vscode-azure-iot-edge" --allow-dupe --allow-redirect --skip-save-results `ls *.md`
+
+    - name: GitHub Release
```

Comment:
>name: GitHub Release [](http://example.com/codeflow?start=6&length=20)

please add a condition that this step executes only if the 2 testing steps before have succeeded